declarations/Midjourney.history.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Terms of Service": [
+    {
+      "fetch": "https://docs.midjourney.com/docs/terms-of-service",
+      "select": [
+        "h1",
+        ".content_container"
+      ],
+      "remove": [
+        "#lightTheme",
+        "#darkTheme"
+      ],
+      "executeClientScripts": true,
+      "validUntil": "to-be-determined"
```

Comment:
@emmanuellar as the automated checks show, this is obviously not a valid date.

In this case, I understand that there simply should be any history file since the change is not related to a change in the document itself but a correction of an initial mistake in the targeting. Simply remove the history file 🙂

Reply:
Thank you Matti for your timely intervention